### ISSUE TITLE
Markdown breaking on JavaScript

### ISSUE THREAD

**Issue by wldcordeiro** [sharkdp/bat#183]
If you create a markdown file with this content it'll break with `bat` and output this error, this may be an issue for upstream though reading the backtrace.

https://gist.github.com/wldcordeiro/136a85b2952a5ad91b2a5fecf13857b1

```
thread 'main' panicked at 'Can only call resolve on linked references: ByScope { scope: <source.javascript>, sub_context: None }', /Users/wellingtoncordeiro/.cargo/registry/src/github.com-1ecc6299db9ec823/syntect-2.1.0/src/parsing/syntax_definition.rs:205:18
stack backtrace:
   0: std::sys::unix::backtrace::tracing::imp::unwind_backtrace
   1: std::sys_common::backtrace::print
   2: std::panicking::default_hook::{{closure}}
   3: std::panicking::default_hook
   4: std::panicking::rust_panic_with_hook
   5: std::panicking::begin_panic_fmt
   6: syntect::parsing::syntax_definition::ContextReference::resolve
   7: syntect::parsing::parser::ParseState::push_meta_ops
   8: syntect::parsing::parser::ParseState::parse_line
   9: syntect::easy::HighlightLines::highlight
  10: bat::features::print_files
  11: bat::run
  12: bat::main
  13: std::rt::lang_start::{{closure}}
  14: std::panicking::try::do_call
  15: __rust_maybe_catch_panic
  16: std::rt::lang_start_internal
  17: main
```

**Comment by sharkdp** [comment]
Thank you for reporting this. This is an upstream bug and I think it has recently been closed: https://github.com/sublimehq/Packages/pull/1618

We probably just have to update sublimehq/Packages.

**Comment by sharkdp** [comment]
I just confirmed that this can be fixed by updating sublimehq/Packages. We just have to wait for https://github.com/sublimehq/Packages/pull/1644 to get merged.

**Comment by felipesere** [comment]
sublimehq/Packages#1644 has been merged 😄 
This issue is the only thing holding me back from upgrading from 0.2.3

**Comment by sharkdp** [comment]
Fixed in [bat 0.5](https://github.com/sharkdp/bat/releases/tag/v0.5.0).
